crates/uv/tests/venv.rs

```diff
@@ -161,6 +161,24 @@ fn create_venv_defaults_to_cwd() {
     context.venv.assert(predicates::path::is_dir());
 }
 
+#[test]
+fn create_venv_ignores_virtual_env_variable() {
+    let context = VenvTestContext::new(&["3.12"]);
+    // We shouldn't care if `VIRTUAL_ENV` is set to an non-existant directory
```

Comment:
Can you add a line stating why we don't care?

Reply:
Definitely